tools/tests/test_camera_stream_config.py

```
@@ -158,4 +158,122 @@ async def run():
+
+    The URL alone cannot tell the two apart, so the endpoint has to key off what
+    initialization actually connected to; guessing 18554 here pointed HLS at a
+    port nothing was listening on.
+    """
+    import asyncio
+
+    cam = _camera(go2rtc_url="http://127.0.0.1:11984")
+    cam._go2rtc_server_url = "http://127.0.0.1:11984/"
+    cam._go2rtc_is_ha_managed = False
+
+    async def run():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `run`

(ANN202)